lib/messagebird.js

```diff
@@ -743,6 +743,102 @@ module.exports = function (accessKey, timeout, features) {
       }
     },
 
+    voice: {
+
+      webhooks: {
+        /**
+         * Creates a new webhook.
+         *
+         * @param {Object} params
+         * @param {Function} callback
+         * @return {void}
+         */
+          create: function (params, callback) {
+            httpRequest({
+              hostname: VOICE_ENDPOINT,
+              method: 'POST',
+              path: '/webhooks',
+              params: params,
+            }, callback);
+          },
+
+        /**
+         * Retrieves an existing webhook by id.
+         *
+         * @param {String} id
+         * @param {Function} callback
+         * @return {void}
+         */
+          read: function (id, callback) {
+            httpRequest({
+              hostname: VOICE_ENDPOINT,
+              method: 'GET',
+              path: '/webhooks/' + id,
+            }, callback);
+          },
+
+        /**
+         * Updates a webhook.
+         *
+         * @param {String} id
+         * @param {Object} params
+         * @param {Function} callback
+         * @return {void}
+         */
+        update: function (id, params, callback) {
+          httpRequest({
+            hostname: VOICE_ENDPOINT,
+            method: 'PUT',
+            path: '/webhooks/' + id,
+            params: params
+          }, callback);
+        },
+
+        /**
+         * Retrieves a list of webhooks.
+         *
+         * @param {Number} limit
+         * @param {Number} offset
+         * @param {Function} callback
+         * @return void
+         */
+        list: function (perPage, currentPage, callback) {
+          var params = null;
+
+          if (typeof callback === 'function') {
+            params = {
+              perPage: perPage,
+              currentPage: currentPage
+            };
+          } else {
+            callback = perPage;
```

Comment:
that looks a little strange why are you assigning a number value to a variable called callback?